### PR TITLE
Support custom start

### PR DESCRIPTION
**Pull request by rkyle35242**
|Before|After (angleStart = 90)|
|---|---|
|image|image|

### CODE REVIEW

**Review by calebmackdavenport — APPROVED** [review]

File: src/SegmentedArc.js [hunk 1]
```diff
@@ -41,7 +42,7 @@ export const SegmentedArc = ({
   const totalSpacing = totalSpaces * spaceBetweenSegments;
 
   const arcSegmentDegree = (arcDegree - totalSpacing) / totalArcs;
-  const arcsStart = 90 - arcDegree / 2;
+  const arcsStart = angleStart || 90 - arcDegree / 2;
```
Comment (calebmackdavenport) [inline src/SegmentedArc.js:45]: Think it's worth documenting any guidance on realistic values here? Or does it feel pretty obvious. Code makes sense, just thinking aloud

Comment (rkyle35242) [reply src/SegmentedArc.js:45]: Example of 720 angleStart:

image

File: src/__tests__/__snapshots__/SegmentedArc.spec.js.snap [hunk 2]
```diff
@@ -742,6 +742,132 @@ exports[`SegmentedArc renders with middle content 1`] = `
...
+      preserveAspectRatio="xMidYMid meet"
+      width={240}
+    >
+      <Context.Provider
+        value={
+          {
+            "animationDuration": 1000,
+            "arcAnimatedValue": 0,
+            "arcSegmentDegree": 43.5,
+            "arcsStart": 420,
```
Comment (Lein-Haz) [inline src/__tests__/__snapshots__/SegmentedArc.spec.js.snap:759]: 🔥

**Review by Lein-Haz — APPROVED** [review]

File: src/__tests__/SegmentedArc.spec.js [hunk 3]
```diff
@@ -154,4 +154,10 @@ describe('SegmentedArc', () => {
     expect(Animated.timing).toHaveBeenCalledTimes(1);
     expect(Easing.out).toHaveBeenCalledWith(Easing.ease);
   });
+
+  it('sets the arc starting point', () => {
+    props.angleStart = 420;
```
[the remark this answers is not in the document]
Comment (Lein-Haz) [reply src/__tests__/SegmentedArc.spec.js:159]: `*Checks date*`, we missed an opportunity by 5 days

File: src/SegmentedArc.js (hunk 1 above)
Comment (jkhusanov) [reply src/SegmentedArc.js:45]: What do you think about `arcStartDegree`? Also, would you be able to update docs too along with the previously added `arcDegreeScale` property? Could be done separately. https://github.com/shipt/segmented-arc-for-react-native?tab=readme-ov-file#-props

**Review by rykugur — APPROVED** [review]

**Comment by rkyle35242** [comment]
I'm actually going to close this. The use case I had in mind is no longer valid and this doesn't work in all cases that I'd like it to. 

That being said, I will take some time to update the README to support the new scale property for segments

**Pull request closed without merging**